File: Android/sources/com/google/android/gms/ads/identifier/zze.java
package com.google.android.gms.ads.identifier;

import java.io.IOException;
import java.net.HttpURLConnection;
import java.net.URL;

public final class zze {
    public static final void zza(String str) {
        HttpURLConnection httpURLConnection;
        try {
            httpURLConnection = (HttpURLConnection) new URL(str).openConnection();
            if (httpURLConnection.getResponseCode() >= 200) {
            }
            httpURLConnection.disconnect();
        } catch (IndexOutOfBoundsException e) {
            e.getMessage();
        } catch (IOException | RuntimeException e2) {
            e2.getMessage();
        } catch (Throwable th) {
            throw th;
        }
    }
}

File: IOS/v2/com/google/android/gms/ads/identifier/zze.swift
package com.google.iOS.gms.ads.identifier;.swift
import java.io.IOException;.swift
import java.net.HttpURLConnection;.swift
import java.net.URL;.swift
public final class zze {.swift
public static final void zza(String str) {.swift
HttpURLConnection httpURLConnection;.swift
try {.swift
httpURLConnection = (HttpURLConnection) new URL(str).openConnection();.swift
if (httpURLConnection.getResponseCode() >= 200) {.swift
}.swift
httpURLConnection.disconnect();.swift
} catch (IndexOutOfBoundsException e) {.swift
e.getMessage();.swift
} catch (IOException | RuntimeException e2) {.swift
e2.getMessage();.swift
} catch (Throwable th) {.swift
throw th;.swift
}.swift
}.swift
}.swift
